docs/tutorials/kubernetes-cluster-in-aws-with-gardener/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create a kubernetes cluster in AWS with Gardener
+level: beginner
+category: Getting Started
+reviewer:
+status:
+last_reviewed:
+scope: app-developer
+---
+
+### Prerequisites
+
+-   You need an AWS account.
+-   You have access to the Gardener dashboard and have permissions to create projects.
+
+1. Go to the Gardener dashboard and create a *Project*.
+
+    <img src="images/new_gardener_project.jpg">
+
+
+2. To copy the policy for AWS from the Gardener dashboard, choose *Secrets*, click on the help icon <img src="images/help_icon.jpg"> for AWS secrets, and choose copy <img src="images/copy_icon.png">.
+
+    <img src="images/gardener_copy_policy.jpg">
+
+3. To [create a new policy](https://console.aws.amazon.com/iam/home?#/policies) in AWS, paste the policy that you copied from the Gardener dashboard to this custom policy.
+
+    <img src="images/create_policy.jpg">
+
+    <img src="images/review_policy.jpg">
+
+4. [Create a new technical user](https://console.aws.amazon.com/iam/home?#/users$new?step=details).
+
+    <img src="images/adduser.jpg">
+
+    <img src="images/attachpolicy.jpg">
+
+    <img src="images/finishuser.jpg">
+
+     > Note: After the user is created, `Access key ID` and `Secret access key` are generated and displayed. Remember to save them. The `Access key ID` is used later to create secrets for Gardener.
+
+
+    <img src="images/savekeys.jpg">
+
+5. On the Gardener dashboard, choose *Secrets* and then the plus sign <img src="images/plus_icon.jpg"> in the AWS frame to add a new AWS secret.
```

Comment:
I think the `<img src="images/plus_icon.jpg">` part is a broken link, the file name currently is `plus_icon.JPG`. Can you rename the file to `plus_icon.jpg` to fix the broken link and to make image extensions consistent (all lowercase)?